### ISSUE TITLE
Raw HTML is shown in docs site page navigation

### ISSUE THREAD

**Issue by adam-rocska** [srmagura/real-cancellable-promise#9]
**Describe the bug**
With the new deploy, apparently the docs' "On this Page" section just broke.

**To Reproduce**
At the time of writing, simply go to https://srmagura.github.io/real-cancellable-promise/#$fetch

If by some miracle it'd be browser specific, I'm using Edge (viva la bing chat)

**Actual Behavior**
Weird anchor HTML elements are rendered literally as text.

**Expected Behavior**
The links should be rendered as links.

**Additional context**
See the following screenshot ¯\\_(ツ)_/¯ 

<img width="1392" alt="Képernyőfotó 2023-04-14 - 17 49 42" src="https://user-images.githubusercontent.com/1914616/232092739-854b1e4a-8766-4193-bc5a-f1c7514e017a.png">

**Comment by srmagura** [comment]
Yeah, I noticed that too. To me, it seems like a bug in [TypeDoc](https://github.com/TypeStrong/typedoc). Would you be interested in reporting the issue to them? Thanks!
